src/RazorSdk/Targets/Sdk.Razor.CurrentVersion.targets

```diff
@@ -637,6 +653,22 @@ Copyright (c) .NET Foundation. All rights reserved.
     </ItemGroup>
   </Target>
 
+  <!-- Transforms the original scoped CSS files into their scoped versions on their designated output paths -->
+  <Target Name="_GenerateScopedCssFiles" Inputs="@(_ScopedCss)" Outputs="@(_ScopedCssOutputs)" BeforeTargets="AfterGenerateScopedCssFiles">
```

Comment:
Note to self: If we decide on this approach, we should rename the "`_`" prefix on some of these names